PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -85,6 +86,12 @@ struct StoreAppConditionalView: View {
 
     @State private var cache = DataCache.instance
 
+    func waitForIconLoad() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 6) {
```

Comment:
This should be switched out to Swift 5.5's concurrency async and await instead of DispatchQueue. You can see an example of how to do a delay after x seconds here: https://stackoverflow.com/questions/59682446/how-to-trigger-an-action-after-x-seconds-in-swiftui